packages/legend-extension-dsl-persistence/src/components/studio/DSLPersistence_LegendStudioPlugin.tsx

```diff
@@ -59,19 +68,22 @@ export class DSLPersistence_LegendStudioPlugin
   override getExtraRequiredDocumentationKeys(): string[] {
     return [
       DSL_PERSISTENCE_LEGEND_STUDIO_DOCUMENTATION_KEY.GRAMMAR_ELEMENT_PERSISTENCE,
+      DSL_PERSISTENCE_LEGEND_STUDIO_DOCUMENTATION_KEY.GRAMMAR_ELEMENT_PERSISTENCE_CONTEXT,
```

Comment:
When you do this, you would introduce a documentation key to be verified at runtime if the corresponding documentation is missing. Please open a corresponding PR on `finos/legend` to add the documentation key - see https://github.com/finos/legend/pull/515 for example

Reply:
Thank you so much!